contentpool_client.module

```diff
@@ -45,6 +48,121 @@ function contentpool_client_form_remote_edit_form_alter(&$form, FormStateInterfa
       ],
     ],
   ];
+
+  $filter_fields = [];
+  $treeselect_filters = contentpool_client_get_treeselect_filters($remote);
+  $current_filter = $remote->getThirdPartySetting('contentpool_client', 'filter', []);
+  foreach ($treeselect_filters as $field => $filter) {
```

Comment:
This cannot work when loading the form the first time, as the connection settings are not stored yet? Please move the UI to a new tab, "Replication filter". The tab should only appear when the remote is a contentpool.

Reply:
it just doesn't show the filter in that case.